src/compiler/objective_c_plugin.cc

```diff
@@ -82,10 +81,12 @@ class ObjectiveCGrpcGenerator : public grpc::protobuf::compiler::CodeGenerator {
         declarations += grpc_objective_c_generator::GetHeader(service);
       }
 
+      ::grpc::string nonNullBegin = "\nNS_ASSUME_NONNULL_BEGIN\n\n";
```

Comment:
Small nit that I just realized: make these two `static const ::grpc::string kNonNullBegin`?